Dockerfile

```diff
@@ -0,0 +1,32 @@
+#
+# Stage: Base environment
+#
+FROM alpine:3.11.5 AS base
```

Comment:
the default `USER` is `root`; can we use `nobody`?
```
$ docker run -it alpine:3.11.5 sh
/ # whoami
root
/ # grep -r nobody /etc/passwd 
nobody:x:65534:65534:nobody:/:/sbin/nologin
```
@erkannt ?

Reply:
Definitely good practice. Will be something that might have to be adjusted as the container actually starts doing things.

Reply:
Like https://github.com/libero/article-page/pull/2/files#diff-3254677a7917c6c01f55212f86c57fbfR76?

Reply:
https://github.com/libero/article-page/pull/1/commits/e89b2f7d7de1cfe34cf1832e3df16813b04ab836